stl/inc/xhash

```diff
@@ -2015,7 +2015,7 @@ _NODISCARD bool _Hash_equal(const _Hash<_Traits>& _Left, const _Hash<_Traits>& _
             // look for element with equivalent key
             const auto& _Keyval = _Traits::_Kfn(_LVal);
             const auto _Next2   = _Right._Find_last(_Keyval, _Right._Traitsobj(_Keyval))._Duplicate;
-            if (!(static_cast<bool>(_Next2) && _Traits::_Nonkfn(_LVal) == _Traits::_Nonkfn(_Next2->_Myval))) {
+            if (!(static_cast<bool>(_Next2) && _LVal == _Next2->_Myval)) {

```

Comment:
A [GH search](https://github.com/search?q=repo%3Amicrosoft%2FSTL%20_Nonkfn&type=code) suggests that this is the only place that `_Nonkfn` was used. Maybe we should remove it entirely?

Reply:
This can definitely be dropped, no ABI impact. Good catch!